### PR TITLE
Add alacritty

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=alacritty CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/alacritty.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  depends_on 'harfbuzz' # R
+  depends_on 'libxcursor' # R
+  depends_on 'libxi' # R
+
+  def self.build
+    system 'CARGO_INCREMENTAL=0 cargo build --release'
+  end
+
+  def self.install
+    FileUtils.install 'target/release/alacritty', "#{CREW_DEST_PREFIX}/bin/alacritty", mode: 0o755
```
Comment (uberhacker) [inline packages/alacritty.rb:44]: Does `FileUtils.install` work without creating directories first now?

Comment (satmandu) [reply packages/alacritty.rb:44]: Sigh.... Not until that PR gets merged...

Comment (satmandu) [reply packages/alacritty.rb:50]: Ok, making those dirs now...

**Review by uberhacker — APPROVED** [review]

File: packages/alacritty.rb (hunk 1 above)
Comment (satmandu) [reply packages/alacritty.rb:50]: And that the ruby fileutils PR just got merged!

I guess we're waiting until ruby 3.2.2 for that though...

**Pull request merged**